js/packages/web/src/views/packCreate/components/Sidebar/data.ts

```diff
@@ -9,7 +9,7 @@ export const STEPS_TITLES: Record<CreatePackSteps, string> = {
   [CreatePackSteps.ReviewAndMint]: 'Review & Mint',
 };
 
-export const CONTINUE_TITLES: Record<CreatePackSteps, string> = {
+export const CONTINUE_TITLES: Record<number, string> = {
```

Comment:
Not sure about this one. Why do we need to generalize the type to a number when it is really more specifically a particular enum?

Reply:
enums values (the way we defined the enum here) are numbers. typescript fails without this.

Reply:
the enum is the same as `{ first: 0, second: 1, third: 2 }`, hence the number type

Reply:
OK it's fine for now. We should probably use `StringUnion`s anyways which are basically a superset of enums.